agent/bench-scripts/pbench-linpack

```diff
@@ -113,24 +117,49 @@ while true; do
 done
 verify_common_bench_script_options ${tool_group} ${sysinfo}
 
-ver="$(pbench-config version ${benchmark})"
-if [[ -z "${ver}" ]]; then
-	error_log "${script_name}: package version is missing in config file"
-	exit 1
-fi
-if [[ -z "${linpack_dir}" ]]; then
-	linpack_dir="/usr/local/${script_name}-${ver}/benchmarks/linpack"
-	linpack_dir_kind="default"
-else
-	linpack_dir_kind="provided"
+function pre_check {
+	# Invoke the linpack driver to perform a pre-check that it will be able to
+	# execute the benchmark. The expected version is the first argument, and the
+	# second is the expected directory prefix for the linpack installation.
+	local ver=${1}
+	local install_prefix_arg=${2}
+
+	${pbench_bin}/bench-scripts/driver/linpack --pre-check ${ver} ${install_prefix_arg}
```

Comment:
The driver option is `--pre-check-only`; `getopt` allows abbreviation, but is that really what we want here?

Reply:
Nice catch, this is a bug.